torch/_prims_common/__init__.py

```diff
@@ -1465,24 +1465,27 @@ def set_correction(
     return correction
 
 
+def compute_required_storage_length(
+    shape: ShapeType, strides: StrideType, storage_offset: int
+) -> int:
+    # Short-circuits if the shape has no elements
```

Comment:
Nice comments. Add a comment for what this helper does and how it's intended to be used, too, please